flake-modules/dev/list-plugins.py

```diff
@@ -65,7 +66,7 @@ def __str__(self) -> str:
                 kind_icon = "\033[92m" + " "
             case _:
                 assert False
-        deprecation_icon: str = "⚠️" if self.dep_warnings else "  "
+        deprecation_icon: str = " " if self.dep_warnings else "  "
```

Comment:
Is this switching from a unicode icon to a nerdfont icon? My browser font can't render the new icon, although it looks fine in the terminal :slightly_smiling_face: 

<img width="150" src="https://github.com/user-attachments/assets/e1db2fcc-1029-48d9-a780-b5d489e5bc2e">

Seems like an unrelated change, though I'm not against it.

Reply:
Perhaps all icons ought to be defined as CONSTANTS at the top of the file, alongside QUESTION_MARK?

I think cleaning up and/or improving icons is orthogonal to the main goal of this PR, but I don't mind sneaking small related changes in through larger PRs if it reduces overall friction to get trivial changes merged.

Might be better in a dedicated commit? Although it is a small enough change it could be lumped in as one commit, maybe just adding a footnote to the commit message.

Reply:
On my mac, I could see the icon when testing. On my linux desktop, for some reason the icon wouldn't show. So I grabbed the latest nerd font because it threw me off when testing on my desktop that I couldn't see the filter working. I can break off the icon and output formatting changes into a separate commit.

Reply:
> On my linux desktop, for some reason the icon wouldn't show.

I haven't looked up the codepoint, but I'd guess it's a unicode emoticon, so you need an emoji font to view it.

IIRC I have [Noto Colour Emoji](https://fonts.google.com/noto/specimen/Noto+Color+Emoji/glyphs) installed system wide, but only have a nerdfont available when in my terminal.

>I can break off the icon and output formatting changes into a separate commit.

Up to you. I wouldn't stress too much over it. Adding the change to the commit message would also be fine, for example.

🙂

Reply:
> I haven't looked up the codepoint, but I'd guess it's a unicode emoticon, so you need an emoji font to view it.
> 
> IIRC I have [Noto Colour Emoji](https://fonts.google.com/noto/specimen/Noto+Color+Emoji/glyphs) installed system wide, but only have a nerdfont available when in my terminal.

Supposedly, I have it installed... https://github.com/khaneliman/khanelinix/blob/c202a44aa74b4a1b2035c232dcc0f86917265f29/modules/shared/system/fonts/default.nix#L32. But, I see it under an /share/x11 folder for some reason in /run/currentsystem and can't find the linked fonts folder...

Reply:
Yeah, I see that I can view the nerd fonts in alacritty but they aren't showing in kitty. So I just pushed the spacing fix for aligning the table columns with mixed output.

Reply:
> Yeah, I see that I can view the nerd fonts in alacritty but they aren't showing in kitty.

For kitty you might find this interesting:

https://github.com/MattSturgeon/nix-config/blob/8645b1687eda188fcc1c656ce0825315b8ef663d/modules/home-manager/kitty.nix#L58-L64

?

Reply:
Sweet, that looks like it was similar to what I found in their documentation. I'll try it out real quick.

Reply:
Worked for me! Thanks!